pkgs/top-level/all-packages.nix

```diff
@@ -6222,6 +6222,12 @@ with pkgs;
 
   dotnetPackages = recurseIntoAttrs (callPackage ./dotnet-packages.nix {});
 
+  gopro-tool = callPackage ../tools/video/gopro-tool {
+    vlc = vlc.overrideAttrs (old: {
+      buildInputs = old.buildInputs ++ [ x264 ];
```

Comment:
Why is this necessary?

Reply:
The default vlc pkg in nixpkgs doesn't seem to have the x264 module which this script relies on.  Should I do this differently?